index.js

```diff
@@ -306,6 +306,27 @@ exports.isValidPrivate = function (privateKey) {
 }
 
 /**
+ * Checks if the public key satisfies the rules of the curve secp256k1
+ * and the requirements of Ethereum.
+ * @method isValidPublic
+ * @param {Buffer} publicKey The two points of an uncompressed key, unless sanitize is enabled
+ * @param {Boolean} [sanitize=false] Accept public keys in other formats
+ * @return {Boolean}
+ */
+exports.isValidPublic = function(publicKey, sanitize) {
+  if (publicKey.length === 64) {
+    // Convert to SEC1 for secp256k1
+    return secp256k1.publicKeyVerify(Buffer.concat([ new Buffer('04', 'hex'), publicKey ]))
```

Comment:
@fanatid can you review this?

SEC1 2.0 (http://www.secg.org/sec1-v2.pdf) only lists 04 for uncompressed keys, but the secp256k1 code has 06 and 07 too.